### ISSUE TITLE
Spectra Loading 

### ISSUE THREAD

**Issue by tditfe** [NFDI4Chem/nmrxiv#999]
**Describe the bug**
Not all the Uploaded Spectra of a Molceule show up in an already public project

**To Reproduce**
1. Go to Dashboard
2. Click on a Project (status Public)
3. Click on Sample 
4. Click on Spectra 
5. Wait for loading to finish
6. Click on the three dots on the right side to open Spectra Overview
7. See Error (missing spectra)

**Expected behavior**
All the uploaded spectra (Proton, Carbon, 2D, etc) should be selectable. (They dont  show up even after reloading multiple times)

**Screenshots**
![Screenshot 2024-01-08 104407](https://github.com/NFDI4Chem/nmrxiv/assets/116734554/9c9d99e1-7d54-4949-b908-138b413eb86c)
![Screenshot 2024-01-08 104545](https://github.com/NFDI4Chem/nmrxiv/assets/116734554/696b084d-a003-4f22-9a13-5857e517d6be)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro;  Version 10,0,22631 Build 22631
 - Browser: Google Chrome Version 120.0.6099.200 (Offizieller Build) (64-Bit)

**Additional Info**

The Spectra show up as intended if I try to access them via "Browse Projects" and not via the Dashboard

**Comment by tilfischer** [comment]
I would like to add that I am pretty sure that the spectrum of that sample loaded before. 

There are also other but somehow similar issue, which just happen sometimes. I didn't grasp in which situation it happens. If one navigates through the project page to the sample page, scrolls down and NMRium loads to display spectra, there are not all spectra shown but only some. I tried to reload the page, but this didn't help. Then I tried it again about half a hour later and it showed all spectra again. 

I observed this serveral times already.

Best,
Tillmann
